nomad/local/integration-tests.nomad

```diff
@@ -147,8 +147,8 @@ job "integration-tests" {
 
   group "python-integration-tests" {
     restart {
-      # I guess I can let it try 2x
-      attempts = 1
+      # Make this a one-shot job. Absolute worst case, Buildkite reruns it for us.
+      attempts = 0
```

Comment:
This turned out to be a bad idea, since my timeout for `make test-integration` would have to account for the extra runtime of a potential retry.